travis/Dockerfile-travis-watch-build

```diff
@@ -0,0 +1,29 @@
+FROM node:6.2
+MAINTAINER ODL DevOps <mitx-devops@mit.edu>
+
+# this dockerfile builds the hub image for the watch container
+# we don't use this directly, instead we push to docker-hub, and
+# then Dockerfile-travis-watch uses that pushed image to bootstrap
+# itself
+
+RUN apt-get update && apt-get install libelf1
+
+RUN mkdir /src
+
+WORKDIR /src
+
+RUN adduser --disabled-password --gecos "" mitodl
+
+RUN chown -R mitodl:mitodl /src
+
+USER mitodl
+
+COPY package.json /src
```

Comment:
Yeah, we add the package manager stuff separately from the source code to get faster builds for development (in our other dockerfiles, like `Dockerfile-node`, for instance).

Here it won't make much of a difference, since we're only using this dockerfile to build the image we push to docker hub, so it will be rebuilt only occasionally. 

I'm fine either way, I think the deciding factor is readability.